src/test/java/pages/booking/BookingHomePage.java

```diff
@@ -75,19 +76,29 @@ public void setNumberOfRooms(int number) {
 
     public void clickOnRegisterButton(){
         driver.findElement(By.xpath(REGISTER_BUTTON)).click();
-
     }
+
     public WebElement getCurrencyField() {
         return driver.findElement(By.xpath(CURRENCY_BUTTON));
     }
+
     public String getToolTipText() {
         return driver.findElement(By.xpath(TOOLTIP_TEXT)).getText();
     }
 
-
     public WebElement getLanguageField() {
         return driver.findElement(By.xpath(LANGUAGE_BUTTON));
+    }
+
+    public String getTextAccountTitle() {
+        return String.format(driver.findElement(By.xpath(ACCOUNT_TITLE)).getText());
+    }
 
+    public void closePopup(){
+        driver.findElement(By.xpath(CLOSE_POPUP_BUTTON)).click();
     }
 
+    public BookingHomePage(WebDriver driver) {
```

Comment:
just create Driver object

Reply:
Error from java: constructor BookingHomePage in class pages.booking.BookingHomePage cannot be applied to given types;